core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsIT.java

```diff
@@ -490,7 +491,8 @@ public void testAckedIndexing() throws Exception {
                                 logger.trace("[{}] indexing id [{}] through node [{}] targeting shard [{}]", name, id, node, shard);
                                 IndexResponse response =
                                         client.prepareIndex("test", "type", id).setSource("{}").setTimeout(timeout).get(timeout);
-                                assertTrue("doc [" + id + "] should have been created", response.isCreated());
+                                assertThat("doc [" + id + "] should have been created", response.getOperation(),
```

Comment:
I don't think the explanation text is important any more now that the error message will be something like `expected [created] but got [another thing]`.